.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,8 +1,8 @@
 ---
-name: "🐛 Bug report"
+name: "\U0001F41B Bug report"
```

Comment:
Did you mean to change this or was it a side-effect of what you were trying to do?

Reply:
sorry I was supposed to fix this, will do it now

Reply:
for the record, GitHub has a UI to edit templates but it did this, so a bit ropey